src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -165,49 +191,65 @@ JitConstants EltwiseKernel_blocked_opt::MakeLoadJitConstants(const eltwise_param
-                                                        "(MAKE_VECTOR_TYPE(ACCUMULATOR_TYPE, " + toCodeString(vec_size) + "))" +
-                                                        "(input" + toCodeString(input.index) + "[0])";
+                        const std::string vload_value = "\\\n\t " + temp_vec_type + temp_vec_var + " = " +
+                                                        "(" + temp_vec_type + ")(input" + toCodeString(input.index) + "[0])";
+
+                        jit.AddConstant(MakeJitConstant(vload_name, vload_value));
+                        jit.AddConstant(MakeJitConstant(name, "tmp_a" + toCodeString(op_num) + "_" + toCodeString(input_idx)));
+                    } else if (feature_broadcasting) {
+                        // Load as scalar and broadcast to vector variable
+                        const std::string broadcast_name = "DO_FEATURE_BROADCAST" + toCodeString(op_num) + "_" + toCodeString(input_idx);
+                        std::string broadcast_value = "\\\n\t " + temp_vec_type + " tmp_broadcast" + toCodeString(op_num) + " = " +
+                                                        "(" + temp_vec_type + ")(input" + toCodeString(input.index) +
+                                                        "[GET_INDEX(INPUT, " + toCodeString(input.index) + ", " + idx_order + ")]);";
+
+                        jit.AddConstant(MakeJitConstant(broadcast_name, broadcast_value));
+                        jit.AddConstant(MakeJitConstant(name, "tmp_broadcast" + toCodeString(op_num)));
+                    } else if (spatial_broadcasting) {
+                        // Load as vector. No need to use GET_INDEX: use f_block for raw indexing
+                        const std::string vload_name = "DO_VLOAD" + toCodeString(op_num) + "_" + toCodeString(input_idx);
+                        const std::string vload_value = "\\\n\t " + temp_vec_type + temp_vec_var + " = " +
+                                                        "TO_TYPE(" + temp_vec_type + ", vload" + toCodeString(vec_size) +
+                                                        "(f_block, input" + toCodeString(input.index) + "));";
+
+                        jit.AddConstant(MakeJitConstant(vload_name, vload_value));
+                        jit.AddConstant(MakeJitConstant(name, "tmp_a" + toCodeString(op_num) + "_" + toCodeString(input_idx)));
+                    } else if (full_tensor) {
+                        // Load as vector. Use raw global id to reduce overhead of formatted indexing
+                        const std::string vload_name = "DO_VLOAD" + toCodeString(op_num) + "_" + toCodeString(input_idx);
+                        const std::string vload_value = "\\\n\t " + temp_vec_type + temp_vec_var + " = " +
+                                                        "TO_TYPE(" + temp_vec_type + ", vload" + toCodeString(vec_size) +
+                                                        "(global_id, input" + toCodeString(input.index) + "));";
```

Comment:
* let's make one more const string: `input + toCodeString(input.index)`
* `vloadn` will be handy too

Reply:
Applied